mysql-test/suite/rpl/t/percona_bug_ps3885.test

```diff
@@ -0,0 +1,26 @@
+#
+# PS-3885: Assertion when switching slaves without stopping the slave thread
+#
+
+--let $rpl_topology= 1->2->3
+--source include/rpl_init.inc
+
+-- connection server_1
+SHOW MASTER STATUS;
```

Comment:
I would avoid recording the output from `SHOW MASTER STATUS` - it may show unstable results.
Either wrap this into
```
--disable_result_log
...
--enable_result_log
```
or extract particular values you are interested in using 
```
--let $xxx = query_get_value("SHOW MASTER STATUS", YYY, 1)
```
and then use assertions.